todo_app/app.py

```diff
@@ -70,7 +62,7 @@ def updateCardToDone(i):
 	    	'id': i,
 			'key': os.environ['KEY'],
 			'token' : os.environ['TOKEN'],
-			'idBoard': '6005828032dafa5707bf5dc3',
+			#'idBoard': '6005828032dafa5707bf5dc3',
 			'idList': '6005828032dafa5707bf5dc7',
```

Comment:
Here and in several methods below you've hardcoded the list ids, even though you have them available from the environment variables (i.e. with toDoId, DoingId & DoneId) - this means that the code can only work with your existing board, and isn't fully configured which means the e2e test still isn't using the fresh board completely. We should adjust all of these, e.g. here to:
```
		query = {
	    	        'id': i,
			'key': os.environ['KEY'],
			'token' : os.environ['TOKEN'],
			'idList': doneId,
		}
```

Reply:
Updated